lib/pbench/server/api/resources/query_apis/datasets/table_of_content.py

```diff
@@ -93,9 +91,8 @@ def assemble(self, json_data: JSON, context: CONTEXT) -> JSON:
     def postprocess(self, es_json: JSON, context: CONTEXT) -> JSON:
         """
         Returns a Flask Response containing a JSON object (keyword/value
-        pairs) where each key is the fully qualified name of a keyword
-        (sub-)field (from run-toc documents) and corresponding value
-        is a non-empty value or list of values which appear in that field.
+        pairs) with directories and files each of which is a list of
+        entries describing individual files and directories.
```

Comment:
I'm laughing at my inner pedant, but...since line 94 says "directories and files", I think line 95 should also have the words in the same order.  😆

More seriously, I would reword line 94 a bit, to something like "Returns a Flask Response containing a JSON object (keyword/value pairs) whose values are lists of entries describing individual files and directories."  (The example will make the verbiage clear.)